spec/lib/web_support_ticket_creator_spec.rb

```diff
@@ -0,0 +1,95 @@
+require "rails_helper"
+
+RSpec.describe WebSupportTicketCreator do
+  let(:args) do
+    {
+      requester: {
+        email: "someone@example.com",
+        name: "Someone",
+      },
+      details: "Foo",
+      link: "link",
+      referrer: "Referrer",
+      user_agent: "User agent",
+      javascript_enabled: true,
+    }
+  end
+
+  let(:support_ticket) { WebSupportTicketCreator.new(args) }
+
+  it "should inherit from TicketCreator" do
```

Comment:
I'm not 100% sure this test adds anything. It's not bad, it just feels a bit like this might be done better with shared tests somehow. I'm not sure enough to let it stop me approving, though.